src/Features/CSharp/Portable/ConvertToRawString/ConvertStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -22,24 +23,22 @@
 namespace Microsoft.CodeAnalysis.CSharp.ConvertToRawString;
 
 [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = PredefinedCodeRefactoringProviderNames.ConvertToRawString), Shared]
-internal sealed partial class ConvertStringToRawStringCodeRefactoringProvider : SyntaxEditorBasedCodeRefactoringProvider
+[method: ImportingConstructor]
+[method: SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
+internal sealed partial class ConvertStringToRawStringCodeRefactoringProvider() : SyntaxEditorBasedCodeRefactoringProvider
 {
-    private static readonly BidirectionalMap<ConvertToRawKind, string> s_kindToEquivalenceKeyMap =
-        new(
-        [
-            KeyValuePairUtil.Create(ConvertToRawKind.SingleLine, nameof(ConvertToRawKind.SingleLine)),
-            KeyValuePairUtil.Create(ConvertToRawKind.MultiLineIndented, nameof(ConvertToRawKind.MultiLineIndented)),
-            KeyValuePairUtil.Create(ConvertToRawKind.MultiLineWithoutLeadingWhitespace, nameof(ConvertToRawKind.MultiLineWithoutLeadingWhitespace)),
-        ]);
+    private static readonly string[] s_kindToEquivalenceKeyMap;
 
-    private static readonly ImmutableArray<IConvertStringProvider> s_convertStringProviders = [ConvertRegularStringToRawStringProvider.Instance, ConvertInterpolatedStringToRawStringProvider.Instance];
-
-    [ImportingConstructor]
-    [SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
-    public ConvertStringToRawStringCodeRefactoringProvider()
+    static ConvertStringToRawStringCodeRefactoringProvider()
     {
+        var count = (int)ConvertToRawKind.ContainsEscapedEndOfLineCharacter * 2;
+        s_kindToEquivalenceKeyMap = new string[count];
+        for (var i = 0; i < count; i++)
+            s_kindToEquivalenceKeyMap[i] = i.ToString(CultureInfo.InvariantCulture);
```

Comment:
>i.ToString [](http://example.com/codeflow?start=43&length=10)

Need more coffee, isn't this just putting string numbers in the array, and not the nameof the corresponding conversion?